packages/flutter/lib/src/material/color_scheme.dart

```diff
@@ -303,10 +842,10 @@ class ColorScheme with Diagnosticable {
   ///  * <https://pub.dev/packages/material_color_utilities>, the package
   ///    used to generate the tonal palettes needed for the scheme.
   factory ColorScheme.fromSeed({
-    required Color seedColor,
```

Comment:
The `seedColor` is optional here, so that instead of this:

```dart
final themeData = Theme.of(context);
final colorScheme = themeData.colorScheme;

return Theme(
  data: themeData.copyWith(colorScheme: colorScheme.copyWith(onPrimary: Colors.black),
  child: child,
);
```

you can do this:

```dart
return Theme.merge(ColorScheme.fromSeed(onPrimary: Colors.black), child: child);
```

If `Theme.merge()` becomes a thing, it'd probably be best not to have any required constructor parameters for the relevant classes.